use-the-built-in-sql-client.md

```diff
@@ -148,26 +148,50 @@ $ echo "SHOW TABLES; SELECT * FROM roaches;" | cockroach sql --user=maxroach --h
 +-----------------------+---------------+
 ~~~
 
-In these examples, we show tables and special characters printed with and without the `--pretty` flag. Note that when not using `--pretty`, table rows are printed as tab-separated values, and special characters are escaped; thus, the output is easy to parse by other programs.
+### Toggling pretty output
```

Comment:
nit: we don't use gerunds in the other example titles, so i'd suggest changing this to `Print with or without pretty output`